Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>TISInputSourceID</key>
```

Comment:
Why is this key moved?

This is a risky change. I need to know the consequences of changing `TISInputSourceID`. Will existing user have to reenable Squirrel in system preferences?

Reply:
I think it's for the "input method only" case:
https://github.com/GreenYun/Zhengma-macOS/wiki/003-info-plist

With input modes, an input source ID is defined for each input mode, by convention it should match the InputModeID:
https://github.com/GreenYun/Zhengma-macOS/wiki/003-info-plist#tisinputsourceid

Reply:
> 但如果 ComponentInputModeDict 有多個副本的話，TISInputSourceID 必須挪到根節點下。

文檔沒有這樣的表述。

事實上代碼文檔說明了各個InputMode都會有對應的TISInputSourceID。
只是所有各處的TISInputSourceID都是可選的，不指定會自動生成一個。

Reply:
Let's drop `.Rime` everywhere. So the input method level source ID will be the same as the bundle identifier --This will be the default input source ID if unspecified. See also my comment below.